vispy/visuals/instanced_mesh.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""A MeshVisual Visual that uses the new shader Function."""
```

Comment:
> that uses the new shader Function.

What does that mean? Maybe at least mention "instanced rendering" somewhere?

Reply:
this is a copy paste from the normal mesh visual, good question xD